pack.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 // PackManifestType represents the manifest type used for [Pack].
```

Comment:
Seems like we wouldn't want to make this change

Reply:
I think Sylvia introduced this is for backward compatibility. That is, we should not introduce any breaking changes in `oras-go v2`.

Reply:
I think I was looking at a diff between commits, ignore